### PR TITLE
Merge feature msbuild into rel100

### PR DESCRIPTION
**Pull request by livarcocc**
@eerhardt @Sridhar-MS @brthor @piotrpMSFT

### CODE REVIEW

File: src/Microsoft.DotNet.Archive/project.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-preview3-*",
+  "version": "1.0.0-featmsbuild-*",
```
Comment (brthor) [inline src/Microsoft.DotNet.Archive/project.json]: Should we change these versions?

**Comment by brthor** [comment]
I think we should include a couple more changes here:
 - move versions to preview4 or something greater than preview3. 
 - change the branchinfo.txt Channel to something new, unless we want everyone on rel-1.0.0 to pick this up (maybe we do?)

**Comment by piotrpMSFT** [comment]
version suffix should remain preview3
everyone on rel-1.0.0 should pick this up.

**Comment by piotrpMSFT** [comment]
You will need to prime stage0 in this branch with a feature-msbuild drop...

**Comment by brthor** [comment]
an alternative to priming stage0 is doing a 2 stage check in, temporarily changing channel/version

**Pull request merged**